src/components/remote_control/src/resource_allocation_manager_impl.cc

```diff
@@ -94,6 +94,49 @@ AcquireResult::eType ResourceAllocationManagerImpl::AcquireResource(
   }
 }
 
+bool ResourceAllocationManagerImpl::ReleaseResource(
+    const std::string& module_type, const uint32_t application_id) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  LOG4CXX_DEBUG(logger_, "Release " << module_type << " by " << application_id);
+  AllocatedResources::const_iterator allocation =
+      allocated_resources_.find(module_type);
+  if (allocated_resources_.end() == allocation) {
+    LOG4CXX_DEBUG(logger_, "Resource " << module_type << " is not allocated.");
+    return false;
+  }
+
+  if (application_id != allocation->second) {
```

Comment:
@dev-gh are you sure, that such complicated logic required?
I would just erase and return void. I see that you don't check result code if this method. So why do you need it? 
You can leave logs with warnings, but result code is redundant

Reply:
@dev-gh please leave checks and remove returns. ANd remove boolean result code of function. It not used

Reply:
@LuxoftAKutsan 
Done with dca9f08